### ISSUE TITLE
Errors when sourcing partially built workspace

### ISSUE THREAD

**Issue by paulbovbel** [colcon/colcon-core#71]
When building a partial workspace using the `--packages-up-to` flag, sourcing the workspace will return errors related to the unbuilt packages:

```
$ colcon build --packages-up-to roscpp
$ source install/setup.bash
not found: "/home/pbovbel/bundle_ws/install/rosbash/share/rosbash/package.bash"
not found: "/home/pbovbel/bundle_ws/install/rosboost_cfg/share/rosboost_cfg/package.bash"
not found: "/home/pbovbel/bundle_ws/install/rosclean/share/rosclean/package.bash"
not found: "/home/pbovbel/bundle_ws/install/rosgraph/share/rosgraph/package.bash"
not found: "/home/pbovbel/bundle_ws/install/class_loader/share/class_loader/package.bash"
not found: "/home/pbovbel/bundle_ws/install/rosmaster/share/rosmaster/package.bash"
not found: "/home/pbovbel/bundle_ws/install/rosparam/share/rosparam/package.bash"
not found: "/home/pbovbel/bundle_ws/install/roscreate/share/roscreate/package.bash"
not found: "/home/pbovbel/bundle_ws/install/roscpp_core/share/roscpp_core/package.bash"
...
```

**Comment by dirk-thomas** [comment]
This should be addressed by #70.

**Comment by paulbovbel** [comment]
Thanks!
